src/component/list.js

```diff
@@ -53,9 +53,15 @@ export class List extends Component {
   render() {
     return (
       <ListView
+        ref={component => (this.list = component)}
         dataSource={this.dataSource}
         renderHeader={this.props.renderHeader}
-        renderRow={this.props.renderItem}
+        renderRow={data => {
+          this.props.scrollToEnd
+            ? setTimeout(() => this.list._scrollViewRef.scrollToEnd(), 50)
```

Comment:
Calling a private attribute will likely break once we upgrade react-native-web. Isn't there a public api for the ListView? Also, why call `_scrollViewRef` when it's a ListView not a ScrollView?

Reply:
I agree, I'm looking for an elegant solution to this. There isn't a public API because the `react-native-web` `ListView` is feature-incomplete.